modules/widgets/synergywholesale_servercost.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+use WHMCS\Database\Capsule as DB;
+use WHMCS\Module\AbstractWidget;
+
+class CostWidgetAPI
+{
+    protected $resellerID;
+    protected $apiKey;
+    protected $soap;
+
+    public function __construct($resellerID, $apiKey)
+    {
+        $this->soap = new SoapClient(null, array("location" => "https://api.synergywholesale.com", "uri" => "", 'trace' => true));
+        $this->apiKey = $apiKey;
+        $this->resellerID = $resellerID;
+    }
+
+    public function request($command, $params = [])
+    {
+        $params['resellerID'] = $this->resellerID;
+        $params['apiKey'] = $this->apiKey;
+
+        try {
+            $response = $this->soap->$command($params);
+        } catch (Exception $e) {
+            // Do stuff?
```

Comment:
Potentially in here, you'd want to return a generic error like this:
```php
return (object) ['errorMessage' => $e->getMessage(), 'status' => 'ERR_EXCEPTION']
```

Or similar, to mimic a Synergy API failure so  you can continue to work, since the $response variable will be undefined when returning an will cause an error